Testing-Notes.md

```diff
@@ -0,0 +1,42 @@
+# Testing Notes
+
+## `/login`
+- `"username"` should not be case sensitive. It should be normalised before it is added to the db. <br>
+**Example**: passing `{"username": "Pavol"}` should be normalised to `{"username": "pavol"}`.
+
+- `"username"` should not take non-alphabetical characters and return an error straight away. Caveat: there is something to be said for allowing numerical characters in the username, but this should be done after careful deliberation <br>
+**Example**: passing `{"username": "Pavol."}` currently returns `User does not exist`
+
+## `/signup`
+- As with Login, `"username"` should not take non-alphabetical characters and return an error straight away. Caveat: there is something to be said for allowing numerical characters in the username, but this should be done after careful deliberation <br>
+**Example**: passing `{"username": "Pavol."}` currently gets accepted just fine.
+**Example**: passsing `{"username": "..."}`, also works at the moment.
```

Comment:
We can add a "warning" on the "create account" component, warning users to only use alphanumerical characters when creating new `username`. 
Created a [Trello card](https://trello.com/c/1kTTbPw2/66-display-warning-when-trying-to-create-new-username-with-non-alphanumerical-characters)